### ISSUE TITLE
Consent screen is presented twice and you have to press twice `Allow`

### ISSUE THREAD

**Issue by scorobogaci** [EddyVerbruggen/cordova-plugin-googleplus#678]
Hello guys, using plugin, but it seems that consent popup is presented twice ? any idea on this >
Also, seems that one time the consent screen is present,second time is doing it silently ? Is this the expected behavior ?

Thanks in advance!

Using : 
"@ionic-native/google-plus": "^5.21.5",

```
    async gPlusLogin() {
        const loading = await this.loadingController.create({
            message: 'Please wait...'
        });

        this.presentLoading(loading);

        return this.googlePlus.login({
            'scopes': '',
            'webClientId': 'xxxx',
            'offline': true,
            'prompt': 'consent'
        }).then(user => {
            loading.dismiss();
            return user;
        }, err => {
            loading.dismiss();
            throw new Error(err);
        })
            .catch((ex) => {
                throw new Error(ex);
            });
    }
```

**Comment by scorobogaci** [comment]
What is the scope of silentLogin if anuways second time log in gives you silent login ?
and first time if using , the silent login will not return anything ? what is the scope of this method ?

**Comment by TehFox** [comment]
offline -> False

**Comment by Ng-India99** [comment]
'offline': false

**Comment by MadyAkira** [comment]
this thread talk about the same issue => [https://github.com/EddyVerbruggen/cordova-plugin-googleplus/issues/675](url)

**Comment by scorobogaci** [comment]
> 'offline': false

this does not help with consent screen being presented twice :(

**Comment by mirko77** [comment]
I have the same issue and I need `offline: true`

Why is it presented twice anyway?

**Comment by jayantpaliwal** [comment]
Stackoverflow thread: https://stackoverflow.com/questions/59258202/how-to-switch-from-multiple-google-oauth-consent-screen-to-single-page

**Comment by mirko77** [comment]
> Stackoverflow thread: https://stackoverflow.com/questions/59258202/how-to-switch-from-multiple-google-oauth-consent-screen-to-single-page

That's another thing though, in our app we get the same consent screen twice in a row. 
Both screens are exactly the same, asking for the same permissions.

**Comment by danpillay87** [comment]
@mirko77 agreed these are two separate issues - in our case even one asking for one permission we get the same consent screen twice in a row, the screens however are not identical with the first screen asking for permission - the second sayng "you already have **** access" but still requiring the allow button to be pressed again.

**Comment by mirko77** [comment]
I'm personally experiencing the double consent screen on other apps or
websites, so it might be an issue on Google side

Best,
Mirko
